cluster/migration.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: Migration
+metadata:
+  name: testvm
```

Comment:
I don't see why.  This states which VM is migrating.  A String name for a migration is a strange requirement, but it seems to be what K8S wants for everything.  Keeping them simple.